packages/dart/sshnoports/lib/src/create_at_client_cli.dart

```diff
@@ -5,7 +5,7 @@ import 'package:path/path.dart' as path;
 
 Future<AtClient> createAtClientCli({
   required String atsign,
-  required String atKeysFilePath,
```

Comment:
OnboardingCli knows the defaultAtKeysPath, and will assume that when this param is null.